content/pt-br/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+O CRI é uma interface de plug-in que permite ao kubelet usar uma ampla variedade
+ de tempos de execução de contêiner, sem a necessidade de recompilar 
+ os componentes do cluster.
+
+Você precisa de um
+{{<glossary_tooltip text="tempo de execução de contêiner" term_id="container-runtime">}} em funcionamento em cada nó em seu cluster,
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} possa iniciar
+{{< glossary_tooltip text="Pods" term_id="pod" >}} e seus contêineres.
+
+{{< glossary_definition prepend="The Container Runtime Interface (CRI) is" term_id="container-runtime-interface" length="all" >}}
```

Comment:
This part is rendering really strangely:
> The Container Runtime Interface (CRI) is a interface de execução de contêiner (CRI) é uma API para agentes de execução de contêineres se integrarem com o kubelet em um nó.

The value for `prepend=""` should be translated. It's OK to translate "Container Runtime Interface" as long as it's clear to the reader what this is about, but ideally we should stick to one way of referring to it throughout the document.